### ISSUE TITLE
please return my decency rating

### ISSUE THREAD

**Issue by L35a** [ValveSoftware/Dota2-Gameplay#13063]
### Description

My account https://steamcommunity.com/profiles/76561198293450213/
27.07.2023 there was a decency rating of 10000
31.07.2023 after updating yours, in which you added Politeness, my rating dropped to 114 !!!!
To be honest, I was stunned how it happened, because the photo shows that the report was formed on 27.07.2023, that in the first case, that in the second, there were no other games during breaks
Maybe you can give me back my rating, it's impossible to play solo on such a low rating!

### Screenshots

![10000](https://github.com/ValveSoftware/Dota2-Gameplay/assets/145040185/2b84b308-57a3-4014-99f9-f35cbeb2ea29)
![114](https://github.com/ValveSoftware/Dota2-Gameplay/assets/145040185/12ba6581-0b9f-41df-8688-babdf3bf46f0)

**Comment by TobyFlendersonFromHR** [comment]
No, we won't return your decency rating. In fact, we are just going to ban you for making such a stupid post in a bug-reporting forum. 

Have a nice day!

**Comment by L35a** [comment]
> No, we won't return your decency rating. In fact, we are just going to ban your steam account for making such a stupid post in a bug-reporting forum.
> 
> Have a nice day!

Hello, I really hope for your decency and honest word. If you have not deceived me, then I am waiting for the account to be banned. Just give the answers in advance
1. if it wasn't a mistake, then what was it?
2. I left a complaint on the link provided to me by the steam staff

**Comment by NAlexP** [comment]
> No, we won't return your decency rating. In fact, we are just going to ban your steam account for making such a stupid post in a bug-reporting forum.
> 
> Have a nice day!

Gtfo troll. There's an issue with the behaviour score.


@L35a https://steamcommunity.com/my/gcpd/570/?category=Account&tab=MatchPlayerReportIncoming Post a screenshot of your incoming match reports.

#13000

**Comment by L35a** [comment]
> > No, we won't return your decency rating. In fact, we are just going to ban your steam account for making such a stupid post in a bug-reporting forum.
> > Have a nice day!
> 
> Gtfo troll. There's an issue with the behaviour score.
> 
> @L35a https://steamcommunity.com/my/gcpd/570/?category=Account&tab=MatchPlayerReportIncoming Post a screenshot of your incoming match reports.
> 
> #13000
![10000 !](https://github.com/ValveSoftware/Dota2-Gameplay/assets/145040185/3ee8c156-5325-4bc8-8c1c-7ddd30be610a)
 пожалуйста, вот скрин доказывающий что 10000 было и я их наиграл, а потом все....

**Comment by NAlexP** [comment]
Seems like there's no problem. You had bad score before as well.

**Comment by L35a** [comment]
> Seems like there's no problem. You had bad score before as well.

the problem is that while playing the party, I reached a stable game without reports. And then, not because of my mistake, but because of the mistakes of the developers, I lost the opportunity to play with friends. Apparently you only help famous people, but not ordinary people. Why would I get 10,000 decency and immediately get 111 the next day? Are you serious about this? Without playing a single game!And the photo only proves it once again, after 10000 my rank turned into 61.
Then explain where the number 111 comes from in 2 photos at all? If there is no such number at all ? Probably screwed himself up?

**Comment by NAlexP** [comment]
> > Seems like there's no problem. You had bad score before as well.
> 
> the problem is that while playing the party, I reached a stable game without reports. And then, not because of my mistake, but because of the mistakes of the developers, I lost the opportunity to play with friends. Apparently you only help famous people, but not ordinary people. Why would I get 10,000 decency and immediately get 111 the next day? Are you serious about this? Without playing a single game!And the photo only proves it once again, after 10000 my rank turned into 61. Then explain where the number 111 comes from in 2 photos at all? If there is no such number at all ? Probably screwed himself up?

I'm not volvo or their representative. I am trying to help devs fix this game, so I can enjoy playing it. That means helping idiots like you from time to time as well.

Explain, please, o russian one, why your score goes from 400 to 10k. Was that not also a developers' mistake?

**Comment by L35a** [comment]
> > > Seems like there's no problem. You had bad score before as well.
> > 
> > 
> > the problem is that while playing the party, I reached a stable game without reports. And then, not because of my mistake, but because of the mistakes of the developers, I lost the opportunity to play with friends. Apparently you only help famous people, but not ordinary people. Why would I get 10,000 decency and immediately get 111 the next day? Are you serious about this? Without playing a single game!And the photo only proves it once again, after 10000 my rank turned into 61. Then explain where the number 111 comes from in 2 photos at all? If there is no such number at all ? Probably screwed himself up?
> 
> I'm not volvo or their representative. I am trying to help devs fix this game, so I can enjoy playing it. That means helping idiots like you from time to time as well.
> 
> Explain, please, o russian one, why your score goes from 400 to 10k. Was that not also a developers' mistake?

if we look at this sign, then everything is very clear there, the photo confirms that, I did not receive reports and played normally, if I was just a sick person, I would not have been able to play without reports. Why should I invent something if I tried and played. Do you think I would just write like that? What's the logic, if I'm so bad, then I'll salt this rating tomorrow, isn't that right? And such an error was also encountered by others, especially valv helped to correct this error to famous streamers

**Comment by L35a** [comment]
the fact that this is a mistake, namely, I'm talking about the fact that the rank has changed for the worse, so these are the photos that I sent, they show that there was no 1 game between them, and the rating has miraculously changed, there is no number 111 in the plate, which means this rating came from nowhere! in addition, it is clear that before 10,000 I received few reports, I corrected myself, but after this mistake everything changed, and what does it have to do with me?

**Comment by TobyFlendersonFromHR** [comment]
i don't believe you

**Comment by L35a** [comment]
freaks, there are no other words, how to get out of 1 world now, I get a report from the patrol every 5 games, although damn, look, why do they give it to me? just like the PTS was lost by famous people, everything was returned to them, but now I can't do anything because of you, the reports are just because I'm playing, there is no evidence of my guilt, but the patrol finds them....They just don't exist, I don't become an afc, I don't die on purpose, I don't break my clothes......
